presets/4.3/rc_link/defaults.txt

```diff
@@ -1,77 +1,13 @@
-#$ TITLE: Default 4.3 RC link settings
+#$ TITLE: Reset RC link-feedforward related settings
 #$ FIRMWARE_VERSION: 4.3
 #$ CATEGORY: RC_LINK
 #$ STATUS: OFFICIAL
 #$ KEYWORDS: defaults, reset, rc link, rc_link
 #$ AUTHOR: Betaflight
-#$ DESCRIPTION: Resets all RC Link related parameters to 4.3 defaults
-# Warning: Resets settings for ALL protocols
-# Note: Does NOT reset serial port settings or Rx Link mode (TAER / AETR etc)
+#$ DESCRIPTION: Resets RC_LINK feedforward smoothing/jitter/averaging parameters and RX provider protocol.
 
 set serialrx_provider = SPEK1024
```

Comment:
If this file is about feedforward settings, we don't need this line to be included, since the kind of serial link to be used is irrelevant.